src/components/TeamMemberTasks/TeamMemberTask.jsx

```diff
@@ -188,7 +190,7 @@ const TeamMemberTask = React.memo(
                                 data-testid={`tick-${task.taskName}`}
                               />
                             )}
-                            {canUpdateTask && (
+                            {canUpdateTask || canDeleteTask && (
```

Comment:
I can see the delete icon only after removing canDeleteTask constant here on line 191, which means either the database is not getting updated when assigning the permission or the value is not handled properly at the frontend.
Please find the screenshot below that prints the value of canDeleteTask on console. 
PS: This component shouldn't be loading more than once as it uses React.memo hook. But as we can see, it re-renders 1169 times. This issue could have been caused by a state/prop change in a parent component that this component uses.
![Screenshot (44)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/48998232/9fb4cff1-9c35-4923-b7a2-c8fb2fc5143d)